azuredevops/provider.go

```diff
@@ -74,6 +74,7 @@ func Provider() *schema.Provider {
 			"azuredevops_team_members":                           core.ResourceTeamMembers(),
 			"azuredevops_team_administrators":                    core.ResourceTeamAdministrators(),
 			"azuredevops_serviceendpoint_permissions":            permissions.ResourceServiceEndpointPermissions(),
+			"azuredevops_servicehooks_permissions":               permissions.ResourceServiceHooksPermissions(),
```

Comment:
I would suggest rename `azuredevops_servicehooks_permissions` to `azuredevops_servicehook_permissions`

Reply:
Thanks for the comments @xuzhang3, I have resolved them 👍